socialhome/frontend/stores/applicationStore.js

```diff
@@ -10,6 +10,7 @@ function newApplicationStore() {
         state: {
             isUserAuthenticated: _get(window, ["context", "isUserAuthenticated"], false),
             currentBrowsingProfileId: _get(window, ["context", "currentBrowsingProfileId"]),
+            // TODO: To be removed
```

Comment:
I've been thinking a lot about the stream loading time which is a big usability thing for me. We had quite a big regression on this with the Vue streams since content is not embedded into the page by Django. I would like to re-introduce that so that the initial page load would add the first set of content and then Vue would handle all the fetching after that (load more, switching streams, etc). This would probably cut down quite a bit of seeing the spinner. It's a nice spinner but still I would like to see it less :)

Thus I think the profile info should also stay, no real need to remove it. The only thing we would be adding is an additional network call on the initial request. Of course Vue should update the information as need be, but I'm just talking about the initial load which we can make faster by making Django and Vue work together like now with the profile info.

Reply:
> I would like to re-introduce that so that the initial page load would add the first set of content and then Vue would handle all the fetching after that (load more, switching streams, etc).

So we would have to maintain to sets of templates. It doesn't look like a smart iead to me :/

Reply:
Oh, yes, that's easy/